source/docs/software/getting-started/using-android-studio.rst

```diff
@@ -24,70 +24,35 @@ Now that you have Android Studio installed, you're going to want to use the curr
 Downloading the SDK
 ^^^^^^^^^^^^^^^^^^^
 
-The SDK is publicly released to a GitHub repository every season. The current season's SDK can be found in the `FtcRobotController <https://github.com/FIRST-Tech-Challenge/FtcRobotController>`_ repository.
+The SDK is publicly released to a GitHub repository. The current SDK can be found in the `FtcRobotController <https://github.com/FIRST-Tech-Challenge/FtcRobotController>`_ repository.
 
-Downloading the ZIP
```

Comment:
We should keep the section about downloading the ZIP file; this is a useful option because the ZIP can be distributed offline easily.

Reply:
The ZIP is a massive footgun that I've had to help teams out of countless times.  IMHO, it's much better to occasionally help a team who actually needs the ZIP, than to constantly have to help teams who picked the ZIP and footgunned themselves in various ways.  (Also, re-adding this section would require re-adding other bits, taking more new screenshots, and overall just make this guide way less streamlined.)